cdip_connector/core/schemas.py

```diff
@@ -75,14 +77,7 @@ class Location(BaseModel):
 
 
```

Comment:
I believe I'm referencing some of these properties on the base model for the CameraTrap object. Specifically location and recorded_at.

Reply:
Oh of course. I've pushed an update that I think will retrofit the CameraTrap model.

Reply:
thanks, looks good to me